packages/eslint-plugin-fiori-tools/README.md

```diff
@@ -17,9 +17,24 @@ Pnpm
 
 ## Usage
 
-To consume this module, add @sap-ux/eslint-plugin-fiori-tools to `extend` in your existing eslintrc configuration. You must specify one of the following configurations:
+To consume this module, add @sap-ux/eslint-plugin-fiori-tools plugin to your `eslint.config.mjs`. You must specify one of the following configurations:
 
 - defaultJS: contains rules for JavaScript for both prod and test code from plugin eslint-plugin-fiori-custom 
 - defaultTS: contains rules for typescript and rules for both prod and test code from plugin eslint-plugin-fiori-custom 
 - testcode: contains rules for typescript and rules for test code from plugin eslint-plugin-fiori-custom
-- prodCode: contains rules for typescript and rules for production code from plugin eslint-plugin-fiori-custom
\ No newline at end of file
+- prodCode: contains rules for typescript and rules for production code from plugin eslint-plugin-fiori-custom
```

Comment:
This need to adjusted once this PR https://github.com/SAP/open-ux-tools/pull/3788 is merged